installers/cubeit-installer

```diff
@@ -72,6 +73,7 @@ cat << EOF
   cubeit-installer <base image> <device>
 
     -b: use btrfs
+    -I(--interactive): use the interactive configuration interface
```

Comment:
-i would be better for this. We currently only use lower case, or long form parameters. -i is consistent with that.